docx/oxml/numbering.py

```diff
@@ -159,7 +159,7 @@ def get_lvl_from_props(self, p, styles_cache=None):
         except AttributeError:
             return None
 
-    def get_num_for_p(self, p, styles_cache):
+    def get_num_for_p(self, p, styles_cache, append_suffix=True):
```

Comment:
It would be helpful to include in docstring what does this flag do

Reply:
Ok, will add docstring explenation